.envrc

```diff
@@ -23,9 +23,14 @@ export REACT_APP_S3_LOCAL_URL='http://localhost:4569'
 export REACT_APP_S3_DOCUMENTS_BUCKET='local-uploads'
 export DATABASE_URL='postgresql://postgres:shhhsecret@localhost:5432/postgres?schema=public&connection_limit=5' # pragma: allowlist secret
 export EMAILER_MODE='LOCAL'
+
 export SES_SOURCE_EMAIL_ADDRESS='dev@localhost'
 export SES_REVIEW_TEAM_EMAIL_ADDRESSES='mc-review-qa@truss.works'
 export SES_RATES_EMAIL_ADDRESSES='mc-rates-review-qa@truss.works'
+export SES_REVIEW_HELP_EMAIL_ADDRESS='MCOGDMCOActions@cms.hhs.gov'
```

Comment:
Shouldn't these be mc-review-qa-help-email@truss.works ? I thought we were only using values here that are overridden in higher environments

Reply:
I'm down to change it, since we don't send emails to any of these addresses I thought it was OK to have them but I don't love that they are checked in even, really. I'm going to merge and we can revisit this after the submission tomorrow.